GitVersion.yml

```diff
@@ -0,0 +1,6 @@
+mode: ContinuousDeployment
```

Comment:
You can ignore it, as it is not really used, I copied from my solutions, what it does if you would have main branch, it would version your package/dll with -rc.X, where X is being n commits from your starting point, or last version... but since it's only tag [commit] everything is ignored